Lib/test/typed_dict.py

```diff
@@ -0,0 +1,19 @@
+"""Script used to test `get_type_hints()` on a class formed by inheriting
+a `TypedDict` with postponed annotations, for bpo-41249, pull request #27017.
```

Comment:
bug number is enough, PR numbers are sometimes confusing because Python's gone through 3 or more version control systems at this point ;)

```suggestion
a `TypedDict` with postponed annotations, for bpo-41249.
```